content/6-design-tools.md

```diff
@@ -29,82 +29,135 @@ preview: tools-preview.jpg
+### Illustrations and Icons
+
+Ready-to-use visual elements that can enhance your project's design:
+
+- [Open Peeps](https://www.openpeeps.com/){:target="_blank"}
+- [Health Icons](https://healthicons.org){:target="_blank"}
+- [Bitcoin Icons](https://bitcoinicons.com/){:target="_blank"}
+
+## Accessibility Tools
 
-Several GUI tools can make Git more approachable for designers who aren't comfortable with the command line:
+Accessibility isn't just a feature - it's a fundamental aspect of good design that should be considered from the start of any project. This is especially important in open source, where our work should be usable by everyone, regardless of their abilities.
 
-1. [GitKraken](https://www.gitkraken.com/): A cross-platform Git client with a user-friendly interface.
-2. [SourceTree](https://www.sourcetreeapp.com/): Another popular Git client with a visual interface for managing repositories.
+Helpful accessibility tools include:
 
-## Collaboration Platforms
+- [WAVE](https://wave.webaim.org/){:target="_blank"}: Evaluate web content for accessibility issues
+- [Contrast Checker](https://webaim.org/resources/contrastchecker/){:target="_blank"}: Ensure your color choices meet accessibility standards
+- [Axe](https://www.deque.com/axe/){:target="_blank"}: Automated accessibility testing
+- [The A11Y Project](https://www.a11yproject.com/){:target="_blank"}: Resources and tutorials for learning about accessibility
 
-Effective collaboration is key in open source projects. While many projects use GitHub for both code and design collaboration, there are other platforms that can enhance the design workflow:
+Remember that accessibility is especially crucial in open source projects, as it aligns with the core values of inclusivity and collaboration. Even developer tools and terminal-based applications should be designed with accessibility in mind.
 
-1. [Penpot](https://penpot.app/): An open source design and prototyping platform that's gaining popularity in the open source community.
-2. [Figma](https://www.figma.com/): While not open source itself, Figma offers a free tier and is widely used in open source projects due to its robust collaboration features.
+## Project Management and Communication
 
-These platforms allow real-time collaboration, making it easier to work with contributors across different time zones and locations.
+- [Miro](https://miro.com/){:target="_blank"}: Digital whiteboard for visual collaboration
+- [Jira](https://www.atlassian.com/software/jira){:target="_blank"}: Project tracking and management
+- [Discord](https://discord.com/){:target="_blank"}: Real-time chat and community building
+- [Matrix](https://matrix.org/){:target="_blank"}: Decentralized communication platform
+- [CryptPad](https://cryptpad.fr/){:target="_blank"}: Privacy-focused collaboration platform
 
```

Comment:
[Excalidraw](https://app.excalidraw.com/) : Excalidraw is an open source alternative to other whiteboard collaboration tools and offers some critical, basic functionality for collaborative note-taking, planning etc.
[Taiga](https://taiga.pm/) : A project management, kanban style 'to-do' list board that is open source. An alternative to Jira.

Reply:
Thank you. I added those tools in [52aee32](https://github.com/GBKS/opendesign.guide/pull/25/commits/52aee3298ca8b2451d833b7b69ed99a5f97d4ba0).